allennlp/common/testing/model_test_case.py

```diff
@@ -92,18 +86,29 @@ def ensure_model_can_train_save_and_load(self, param_file: str, tolerance: float
         for key in model_predictions.keys():
             self.assert_fields_equal(model_predictions[key],
                                      loaded_model_predictions[key],
-                                     tolerance=1e-4,
+                                     tolerance=tolerance,
                                      name=key)
 
         return model, loaded_model
 
-    @staticmethod
-    def assert_fields_equal(field1, field2, tolerance: float = 1e-6, name: str = None) -> None:
+    def assert_fields_equal(self, field1, field2, tolerance: float = 1e-6, name: str = None) -> None:
```

Comment:
a couple of times below you are doing string concatenation on `name`, so maybe empty string is a better default value?

Reply:
Thanks, good catch.  We don't actually need a default value, so I just removed it.